src/main/webapp/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>SAY</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body>  
+    <div>
+      <form>
+        <p>Enter a YouTube video link:</p>
+        <textarea name="link-input"></textarea>
+        </br>
+        <input type="submit" onClick="getChart()"/>
```

Comment:
Are we going to have the YouTube Sentiment Analysis data extracted from the getChart() method? Or would we want onClick to point to a function which grabs the data then puts it into the chart?

Reply:
Talked about in chat - have getChart() call getYoutubeComments()